src/adapters/handlers/nfts.ts

```diff
@@ -33,7 +33,11 @@ export function createNFTsHandler(
     const isLand = params.getBoolean('isLand')
     const isOnRent = params.getBoolean('isOnRent')
     const isWearableHead = params.getBoolean('isWearableHead')
+      ? params.getString('isWearableHead') === 'true'
```

Comment:
This change is needed to know if it needs to be applied in the query as parameter or not
https://github.com/decentraland/nft-server/pull/380/files#diff-d1debe4191e7ee45f023c53cecc0a2d6c05c4dba8f24d44c3a2c69261102a0bcR310